app/gui/src/dashboard/assets/inbox.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Where did you pick this icon up? Not sure it's coming from the central icon system

Reply:
made it manually... not ideal but i'm just really not sure it fits in with that amount of padding
![image](https://github.com/user-attachments/assets/6a8f37b0-a600-4723-8885-a5c19a222bfb)

Reply:
the padding also means badges might look weird